app/api/validators.py

```diff
@@ -24,19 +24,10 @@ def validate_age_order(
         max_age = int(filtered_ordered_response["max_age"])
 
         if min_age > max_age:
-            confirmation = input(
-                f"Sorry but minimum age ({min_age}) can’t be bigger than maximum age ({max_age}). Would you like to see the result for min age {max_age} and max age {min_age}? (please respond with yes/no): "
+            return (
+                "Sorry but minimum age can’t be bigger than maximum age."
             )
 
-            if confirmation.lower() == "yes":
-                filtered_ordered_response["min_age"] = str(max_age)
-                filtered_ordered_response["max_age"] = str(min_age)
-                return filtered_ordered_response
-            else:
```

Comment:
Curious, is there a reason we're removing this?

Reply:
Yes,
For age order validation the current implementation uses the Python input function to get user confirmation. However, the input function is designed for command-line interfaces and does not work in web-based environments (i.e in our case the react frontend).

Reply:
Would it be possible to make adjustments on the side of UI to make this work?

Reply:
Hi @rmanaem,
I have created a new issue for this over [here](#41).